internal/store/atlas/teams.go

```diff
@@ -38,61 +39,63 @@ type TeamDeleter interface {
 }
 
 type TeamAdder interface {
-	AddUsersToTeam(string, string, []string) (interface{}, error)
+	AddUsersToTeam(string, string, []atlasv2.AddUserToTeam) (*atlasv2.PaginatedApiAppUser, error)
 }
 
 type TeamUserRemover interface {
 	RemoveUserFromTeam(string, string, string) error
 }
 
 type TeamRolesUpdater interface {
-	UpdateProjectTeamRoles(string, string, *atlas.TeamUpdateRoles) ([]atlas.TeamRoles, error)
+	UpdateProjectTeamRoles(string, string, *atlasv2.TeamRole) (*atlasv2.PaginatedTeamRole, error)
 }
 
 // TeamByID encapsulates the logic to manage different cloud providers.
 func (s *Store) TeamByID(orgID, teamID string) (*atlas.Team, error) {
+	// Migration Blocked by incorrect OpenAPI schema
```

Comment:
Jira number?

Reply:
I hadn't created one since it's openAPI schema and want to aggregate them all first, but I can create one ticket for all OpenAPI schema fixes and append it as we go before addressing them together at the end of the project